docs/maui/views/popup-service.md

```diff
@@ -113,17 +111,27 @@ public class MyViewModel : INotifyPropertyChanged
 
     public void DisplayPopup()
     {
-        this.popupService.ShowPopup<NamePopupViewModel>();
+        if (Application.Current?.Windows[0].Page is not Page currentPage)
```

Comment:
This isn't a good approach, `Windows[0]` does mean nothing, is not the active Window, isn't the focused Window ,etc... So we shouldn't use it to grab the current Window or current page implementation.

From the top of my head, I can think in these (possible) solutions:
 - Another approach would be to inject the `INavigation` into VM and pass it into the `ShowPopup` method
 - Shell.Current should return the `Shell` instance for that active Window.

Reply:
Oh great point we can now just use `this.popupService.ShowPopup<NamePopupViewModel>(Shell.Current);`